eng/Publishing.props

```diff
@@ -169,14 +120,13 @@
     but we do produce them in both the VMR and the runtime official build.
   -->
   <PropertyGroup>
-    <_ShouldGenerateProductVersionFiles Condition="'$(DotNetBuildRepo)' == 'true' and '$(DotNetBuildOrchestrator)' == 'true'">true</_ShouldGenerateProductVersionFiles>
-    <_ShouldGenerateProductVersionFiles Condition="'$(DotNetBuildRepo)' != 'true' and '$(DotNetBuildOrchestrator)' != 'true'">true</_ShouldGenerateProductVersionFiles>
+    <ShouldGenerateProductVersionFiles Condition="'$(DotNetBuildOrchestrator)' == 'true'">true</ShouldGenerateProductVersionFiles>
```

Comment:
IIRC we only need that file produced once, i.e. in win-x64.

Reply:
I have the build set up to drive it in the runtime-official pipeline (producing it in the workloads job as that's a little easier).

In the VMR, I'm pretty sure that the productVersion.txt is used downstream (I think aspnetcore might reference it? I'm not 100% positive where I saw it).

Reply:
> In the VMR, I'm pretty sure that the productVersion.txt is used downstream (I think aspnetcore might reference it? I'm not 100% positive where I saw it).

I don't think it does? THere is some logic I can see that looks for the files in arcade, and produces one in aspnecore, but no uses I can see in aspnetcore.

Reply:
Yeah I would surprised if that file is read by aspnetcore in the VMR build.

Reply:
In that case, I'll adjust this to run in win-x64 BuildPass1 in the VMR.